apps/aecore/lib/aecore/naming/name.ex

```diff
@@ -1,81 +1,70 @@
-defmodule Aecore.Naming.NameClaim do
+defmodule Aecore.Naming.Name do
```

Comment:
why is the name changed?

Reply:
We found this name more compatible with epoch node

Reply:
okay